flipperdevices/Flipper-Android-App
# Update remote controls

**Background**

This PR introduces update for remote controls with UI improvements and deeplink integration for local-flipper files

**Changes**

- Animated icons when emulating
- Fix bottom navigation paddings
- Save .ui.json file on flipper
- Fix save api paths usage
- Add Raw DeeplinkContent

**Test plan**

- Enable remote controls in debug options
- Open infrareds list screen
- Press add remote
- Select some brand and complete configuration for some device
- See grid screen, tap button and look at beautiful animation
